src/components/UI/Navbar.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please kindly do not disable linters, fix linters errors helps programmers to write clean codebase 😁 😉